frontend/components/Ranking.vue

```diff
@@ -155,19 +155,19 @@
               <b-col cols="2">
                 <a
                   v-b-tooltip.hover
-                  class="select"
-                  title="Select / Unselect validator"
                   @click="toggleSelected(data.item.stashAddress)"
                 >
                   <font-awesome-icon
                     v-if="data.item.selected"
                     icon="hand-paper"
                     class="selected text-selected"
+                    :title="$t('pages.accounts.remove_from_favorites')"
                   />
                   <font-awesome-icon
                     v-else
                     icon="hand-paper"
                     class="unselected text-secondary"
+                    :title="$t('pages.accounts.add_to_favorites')"
```

Comment:
Should the icon be here `hand-paper`?

Reply:
Is should be a star icon for mobile devices